pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1737,6 +1737,121 @@ var _ = Describe("Manager", func() {
 		})
 	})
 
+	It("executes hotPlugHostDevices", func() {
+		os.Setenv("KUBEVIRT_RESOURCE_NAME_test1", "127.0.0.1")
+		os.Setenv("PCIDEVICE_127_0_0_1", "05EA:Fc:1d.6")
```

Comment:
👍 Let's fix this in a follow-up in the code itself. Env vars should be read in the outmost app layer and then passed via variables or struct members.